lib/collections/addon/components/discover-page/component.ts

```diff
@@ -226,12 +225,6 @@ export default class DiscoverPage extends Component {
         return this.totalPages < maxPages ? this.totalPages : maxPages;
     }
 
-    @computed('currentUser.sessionKey')
-    get searchUrl() {
-        // Pulls SHARE search url from config file.
-        return `${config.OSF.shareSearchUrl}?preference=${this.currentUser.sessionKey}`;
-    }
-
```

Comment:
Is this part of this release? @futa-ikeda

Reply:
This shouldn't be necessary as part of this release, as this is for old SHARE powered searches, which we don't use anymore.